lib/compare_with_wikidata/diff_row.rb

```diff
@@ -64,8 +64,13 @@ def value_cells
         # Expand Wikidata IDs to templates.
         # This expands cases where the whole cell is just a Q value:
         value = v.to_s.sub('http://www.wikidata.org/entity/', '').sub(/^Q(\d+)$/, '{{Q|\\1}}')
-        # This expands cases where the whole cell is a change in Q value:
-        value.sub!(/^Q(\d+)->Q(\d+)$/, '{{Q|\\1}}->{{Q|\\2}}')
+        # This expands cases where the whole cell is a change in Q
+        # value (and either side of the change might be NULL)
+        if value =~ /^(Q\d+|NULL)->(Q\d+|NULL)$/
+          value = Regexp.last_match.captures.map do |c|
+            c == 'NULL' ? 'NULL' : "{{Q|#{c[1..-1]}}}"
```

Comment:
this could probably be simplified by not trimming the leading "Q" — `{{Q|Q42}}` works just as well as `{{Q|42}}`